### PR TITLE
Messing about

### PR DESCRIPTION
**Pull request by matthdsm**

### CODE REVIEW

**Comment by matthdsm** [comment]
This still fails `./gradlew check` with a bunch of errors

File: plugins/nf-nomad/src/main/nextflow/nomad/batch/NomadBatchExecutor.groovy [hunk 1]
```diff
@@ -114,11 +129,15 @@ class NomadExecutor extends Executor implements ExtensionPoint {
...
+    }
+
+    NomadBatchService getBatchService() {
+        return batchService
     }
 
-    NomadExecutor getNomadExecutor() {
-        return nomadExecutor
+    NomadBatchExecutor getNomadBatchExecutor() {
+        return nomaBatchdExecutor
```
Comment (abhi18av) [inline plugins/nf-nomad/src/main/nextflow/nomad/batch/NomadBatchExecutor.groovy:140]:
```suggestion
        return nomadBatchExecutor
```

File: plugins/nf-nomad/src/main/nextflow/nomad/batch/NomadBatchService.groovy [hunk 2]
```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 2021, Microsoft Corp
```
Comment (abhi18av) [inline plugins/nf-nomad/src/main/nextflow/nomad/batch/NomadBatchService.groovy:2]:
```suggestion
 * Copyright 2023 FIXME
```

**Review by abhi18av — APPROVED** [review]
Added minor suggestions. In your case, it's failing probably because of the absence of `_resources/nextflow` as was highlighted here https://github.com/CenterForMedicalGeneticsGhent/nf-nomad/pull/4#discussion_r989364595 


I think it's okay to merge since the changes are about conventions in this PR.

**Comment by matthdsm** [comment]
Thanks for the suggestions!
I'm still getting the following errors:
```
/Users/matdsmet/Projects/nf-nomad/plugins/nf-nomad/src/main/nextflow/nomad/batch/NomadBatchExecutor.groovy: 102: [Static type checking] - Cannot set read-only property: batchService
 @ line 102, column 9.
           batchService = new NomadBatchService(this)
           ^

/Users/matdsmet/Projects/nf-nomad/plugins/nf-nomad/src/main/nextflow/nomad/batch/NomadBatchExecutor.groovy: 104: [Static type checking] - Cannot find matching method io.nomadproject.client.ApiClient#<init>(nextflow.nomad.batch.NomadBatchExecutor). Please check if the declared type is correct and if the method exists.
 @ line 104, column 30.
           nomadBatchExecutor = new ApiClient(this)
                                ^

/Users/matdsmet/Projects/nf-nomad/plugins/nf-nomad/src/main/nextflow/nomad/batch/NomadBatchExecutor.groovy: 104: [Static type checking] - Cannot set read-only property: nomadBatchExecutor
 @ line 104, column 9.
           nomadBatchExecutor = new ApiClient(this)
```

Something about typing errors, but I'm not really sure what to do with them

**Comment by abhi18av** [comment]
I'm merging this prior to the fork-and-repo-port process.

Whatever is failing we'll fix on the upstream master branch.

**Pull request merged**